tests/testthat/test-ui-donnut.R

```diff
@@ -7,4 +7,6 @@ test_that("cXdonut1", {
 
 test_that("cXdonut2", {
     check_ui_test(cXdonnut2())
+
+    warning("Clicking on legend misplace the labels on chart")
```

Comment:
change to fail - and to message 'Legend click results in label placement issues'